internal/perf/monitor_unix.go

```
@@ -333,6 +340,99 @@ func parseRocmSmiLine(header string, line string) *GpuStat {
+					continue
+				}
+				if r == '\\' && inString {
+					escaped = true
+					continue
+				}
+				if r == '"' {
+					inString = !inString
+					continue
+				}
+				if !inString {
+					if r == '{' {
```

golangci-lint (2.12.2)

[error] QF1003: could use tagged switch on r

(staticcheck)